Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Block_URL.yml

```diff
@@ -381,6 +378,21 @@ tasks:
       tsg_id:
         complex:
           root: inputs.TSGID
+      value:
+        complex:
+          root: inputs.URL
+          filters:
+            - - operator: notIn
+                left:
+                  value:
+                    simple: inputs.URL
+                  iscontext: true
+                right:
+                  value:
+                    simple: PrismaSase.CustomURLCategory.list
+                  iscontext: true
+          transformers:
+            - operator: uniq
```

Comment:
I dont know why I cant comment in the right line so this comment refering to task 16 (from line 562 - Use Custom Block Rules?).
I dont see an input with the name inputs.CustomBlockRule

Reply:
Good catch -> this task should be the `Choose to commit and push configuration` instead of this.